src/main/java/com/salesforce/functions/jvm/sdk/data/RecordAccessor.java

```diff
@@ -86,8 +91,9 @@ public interface RecordAccessor {
    *
    * <p>If the field is not a boolean, its value is converted to a {@link String} first, and then
    * parsed with {@link Boolean#parseBoolean(String)}. Effectively, this means that any non-boolean
-   * value except the String "true" (compared case-insensitive) will result in false being returned
-   * by this method.
+   * value that can be converted into a {@link String}, except the String "true" (compared
+   * case-insensitive) will result in false being returned by this method. If the field cannot be
```

Comment:
```suggestion
   * case-insensitively) will result in false being returned by this method. If the field cannot be
```

Reply:
This is in existing docs as well - I will fix this in a separate PR.